Misc/NEWS.d/next/Library/2025-01-10-13-06-54.gh-issue-118761.f8oADD.rst

```diff
@@ -0,0 +1,4 @@
+Improve performance of :func:`base64.b16decode` by a factor ten by
+removing an unnecessary regular expression. Consequently, :mod:`re`
+is no more implicitly available as ``base64.re`` and import time of
+:mod:`base64` is improved by a factor six. Patch by Bénédikt Tran.
```

Comment:
We can then put more detail in the changelog (What's New should be a high level summary):

```suggestion
Improve the performance of :func:`base64.b16decode` by up to ten times
by more efficiently checking the string for hexadecimal digits.
Reduce the import time of :mod:`base64` by up to six times,
by no longer importing :mod:`re`.
Patch by Bénédikt Tran, Chris Markiewicz, and Adam Turner.
```

I'm not sure it is worth mentioning that ``base64.re`` is removed as an attribute -- we haven't mentioned these for the other PRs and trying to access a stdlib module via another one is mildly ridiculous.